draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -267,6 +291,17 @@ and lastly the optional Key Binding JWT.
 In the case that there is no Key Binding JWT, the last element MUST be an empty
 string and the last separating tilde character MUST NOT be omitted.
 
+The serialized format for an SD-JWT-KB extends the SD-JWT format by concatenating a Key Binding JWT.
+
+```
+<Issuer-signed JWT>~<Disclosure 1>~<Disclosure 2>~...~<Disclosure N>~<KB-JWT>
+```
+
+The two formats can be distinguished by the final `~` character that is present
+on an SD-JWT.  A Verifier that expects an SD-JWT MUST verify that its final
+`~`-separated component is empty.  A Verifier that expects an SD-JWT-KB MUST verify
+that its final `~`-separated component is a valid KB-JWT.
```

Comment:
"~-separated" looks rather funky when reading it - I think using the word tilde here would be more good 

```suggestion
tilde-separated component is empty.  A Verifier that expects an SD-JWT-KB MUST verify
that its final tilde-separated component is a valid KB-JWT.
```